airootfs/usr/share/vx-img/flash-image.sh

```diff
@@ -71,6 +71,25 @@ fi
 
 clear
 
+# Detect if this disk already has VotingWorks data on it and copy the machine config
+vg=$(vgscan | sed -s 's/.*"\(.*\)".*/\1/g')
+
+if [[ -n $vg ]]; then
+
+    # Mount the root partition so symlinks are preserved.
+    dir=$(find "/dev/$vg" -name "var")
+    mount "$dir" /mnt
+
+    if [ -d "/mnt/vx/config" ]; then
+        echo "Found /vx/config to copy. Press any key to continue."
+        read -r
+        tar -czvf vx-config.tar.gz /mnt/vx/config
```

Comment:
where does this get stored? is this all a ramdisk by default?

Reply:
yep, just shoved into RAM. That will be a problem if it ever gets too large, but right now it's like, kilobytes.